server/views/layout/patterns.njk

```diff
@@ -14,16 +14,7 @@
-</svg>
+{% include "views/partials/svg-sprite.njk" %}
```

Comment:
and then you can remove the [`.gitkeep`](https://github.com/wellcometrust/wellcomecollection.org/pull/24/commits/23187241c1d509711ec9e508317190ced033d2ac).